fertiscan_pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "fertiscan_datastore"
-version = "1.0.3"
+version = "1.0.4"
 authors = [
   { name="Francois Werbrouck", email="francois.werbrouck@inspection.gc.ca" }
   { name="Kotchikpa Guy-Landry Allagbe" , email = "kotchikpaguy-landry.allagbe@inspection.gc.ca"}
```

Comment:
I think there might be an error. Check suggestion above

Reply:
Wonder how it passed workflows without failing.

Reply:
same here. tagging @SonOfLope so he can be noticed of this weird behavior

Reply:
Thats because it compares with the releases and latest  release was 1.0.2. The errors is that release 1.0.3 doesnt exist. Thats because it shouldve been created when this PR was merged : 
https://github.com/ai-cfia/ailab-datastore/pull/178
Problem is, releases are done when changes are made to folder fertiscan or nachet or datastore but that PR only changed the pyproject and the test folder. Nice edge case guys ! I will update pipeline to handle this case :)